assembly/buffer/index.ts

```diff
@@ -23,6 +23,12 @@ export class Buffer extends Uint8Array {
     return result;
   }
 
+  writeInt8(value: i8, offset: i32 = 0): i32 {
+    if(<u32>offset > this.dataLength) throw new RangeError(E_INDEXOUTOFRANGE);
```

Comment:
Shouldn't this be `offset >= this.dataLength`?

Reply:
Not sure how I missed that. Thanks for the heads up.